rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -57,9 +57,27 @@ bool verify_mself(u32 fd, fs::file const& mself_file)
 	return true;
 }
 
-lv2_fs_mount_point* lv2_fs_object::get_mp(const char* filename)
+lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename)
 {
-	// TODO
+	const auto mp_begin = filename.find_first_not_of('/');
+
+	if (mp_begin + 1)
+	{
+		const auto mp_name = filename.substr(mp_begin, filename.find_first_of('/', mp_begin));
+
+		if (mp_name == "dev_hdd1"sv)
+			return &g_mp_sys_dev_hdd1;
+		if (mp_name.substr(0, 7) == "dev_usb"sv)
```

Comment:
There is `starts_with` now (C++20), but it requires `/std:c++latest` which may not be desirable yet.

Reply:
Not available yet